test/test_numpy_interop.py

```diff
@@ -229,20 +229,20 @@ def test_from_numpy(self, device) -> None:
     def test_ctor_with_numpy_scalar_ctor(self, device) -> None:
         dtypes = [
             np.double,
-            np.float,
+            np.float_,
```

Comment:
I'm not a fan of using `float_`, `np.float64` is more readable and better practice.

Reply:
Comment applies to multiple places.